core/src/mindustry/game/Rules.java

```diff
@@ -38,6 +38,10 @@
     public boolean canGameOver = true;
     /** Whether reactors can explode and damage other blocks. */
     public boolean reactorExplosions = true;
+    /** Whether schematics are allowed */
+    public boolean schematicAllowed = true;
+    /** Whether copying is allowed */
+    public boolean copyAllowed = true;
```

Comment:
I don't think this should be changeable by servers. Copying is very useful, and doesn't have the same issues that schematics may have.

Reply:
I will push a new version with some major changes, so I will close this if it is fine.